IdentityCore/src/util/ios/MSIDAppExtensionUtil.h

```diff
@@ -42,5 +42,6 @@
 + (void)sharedApplicationOpenURL:(nonnull NSURL *)url
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->